medicine.html

```diff
@@ -0,0 +1,228 @@
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="l" value="^l" v-model="filter" autocomplete="off" checked>L
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="m" value="^m" v-model="filter" autocomplete="off" checked>M
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="n" value="^n" v-model="filter" autocomplete="off" checked>N
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="o" value="^o" v-model="filter" autocomplete="off" checked>O
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="p" value="^p" v-model="filter" autocomplete="off" checked>P
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="q" value="^q" v-model="filter" autocomplete="off" checked>Q
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="r" value="^r" v-model="filter" autocomplete="off" checked>R
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="s" value="^s" v-model="filter" autocomplete="off" checked>S
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="t" value="^t" v-model="filter" autocomplete="off" checked>T
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="u" value="^u" v-model="filter" autocomplete="off" checked>U
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="v" value="^v" v-model="filter" autocomplete="off" checked>V
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="w" value="^w" v-model="filter" autocomplete="off" checked>W
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="x" value="^x" v-model="filter" autocomplete="off" checked>X
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="y" value="^y" v-model="filter" autocomplete="off" checked>Y
+                </label>
+                <label class="btn btn-secondary">
+                    <input type="radio" name="options" id="z" value="^z" v-model="filter" autocomplete="off" checked>Z
+                </label>
+            </div>
```

Comment:
Neviem, prečo to nefunguje, a asi by to nebolo také ľahké spraviť. Zatiaľ to zmaž a potom, ak jej to bude vadiť, sa na to pozrieme.